languagetool-language-modules/de/src/main/java/org/languagetool/tokenizers/de/GermanWordTokenizer.java

```diff
@@ -24,7 +24,7 @@
 
 public class GermanWordTokenizer extends WordTokenizer {
   
-  private final String deTokenizingChars = super.getTokenizingCharacters() + "_";
+  private final String deTokenizingChars = super.getTokenizingCharacters() + "_‚";
```

Comment:
Ich bin mir nicht sicher, was diese Änderung bewirken soll. Vlt könntest du das kurz erläutern?

Reply:
Bei dem ergänzten Zeichen handelt es sich um das öffnende einfache Anführungszeichen, das in deutschen Standardtexten verwendet wird. Im Gegensatz zu allen anderen ähnlichen Zeichen (z.B. dem korrespondierenden schließenden Anführungszeichen) wird es nicht in der Liste der Sonderzeichen  (TOKENIZING_CHARACTERS) im WordTokenizer geführt. Zumindest für deutsche Sprache gilt, dass es ein Sonderzeichen ist und auch so behandelt werden muss. Andernfalls wird es wie ein Buchstabe behandelt und dem eigentlichen Word im Token vorangestellt. Das hat übrigens den Seiteneffekt, dass alle mit einem Anführungszeichen beginnenden Worte nicht erkannt werden (PosTag: unknown).  Für die GermanUnpairedBracketsRule hat es zur Folge, dass die einfachen Anführungszeichen nicht definiert werden können, weil das öffnende Zeichen nie als einzelnes Token gehandelt wird.
Daher muss dieses Zeichen (zumindest für deutsche Sprache) zu den Sonderzeichen hinzugefügt werden. Leider bin ich nicht in der Lage zu beurteilen, ob dieses Zeichen in jeder anderen unterstützten Sprache als Sonderzeichen geführt werden darf oder nicht.  Daher habe ich den Weg gewählt, es nur dem GermanWordTokenizer hinzuzufügen. Sollte das Zeichen schlicht vergessen worden sein, wäre es natürlich sauberer, es den TOKENIZING_CHARACTERS im WordTokenizer hinzuzufügen.